build-plugin/sfx_scripts/jenkins-build

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -ex
+#This script is to build the collectd-signalfx plugin in Jenkins. It need the environment variables JOB_NAME and BASE_DIR.
+
+if ! [ -z $JOB_NAME ] && ! [ -z $BASE_DIR ]; then
+  trap "docker rm ${JOB_NAME}" EXIT
+  #Build the packages
+  docker run --privileged --name ${JOB_NAME} -t -v ${BASE_DIR}/${JOB_NAME}/signalfx-collectd-plugin:/opt/signalfx-collectd-plugin -v ${BASE_DIR}/${JOB_NAME}/collectd-build-ubuntu/build-plugin:/opt/collectd-plugin-build-debian -v ${BASE_DIR}/${JOB_NAME}/workspace:/opt/workspace -v ${BASE_DIR}/${JOB_NAME}/result:/opt/result quay.io/signalfuse/collectd-build-ubuntu /opt/collectd-plugin-build-debian/sfx_scripts/cmdseq
+
+  if [ "${BUILD_PUBLISH}" = "True" ] ; then
+        # Upload the packages to Amazon S3 bucket
+        aws s3 rm --recursive s3://collectd-builds-ubuntu/signalfx-collectd-plugin/
+        aws s3 cp --recursive ${BASE_DIR}/${JOB_NAME}/result/ s3://collectd-builds-ubuntu/signalfx-collectd-plugin/
```

Comment:
I think you should use a separate directory (tempdir) like wentao does in his jobs..
Either way you don't want to use BASE_DIR.. that is the PATH outside the jenkins docker image.. you just want /opt/jenkins/workspace/${JOB_NAME}/result/